web/profiles/custom/os2loop/modules/os2loop_messages/src/Helper/Helper.php

```diff
@@ -94,4 +114,33 @@ private function getMessageTemplate(EntityInterface $entity, string $operation)
     return $template;
   }
 
+  /**
+   * Implements hook_form_alter().
+   *
+   * If subscription is enabled on documents/document collections
+   * show notify checkbox on content.
+   */
+  public function formAlter(&$form, $form_id) {
```

Comment:
An `array` type hint on `&$form` might be appropriate here.

Reply:
Added